client/signup/config/flows.js

```diff
@@ -17,6 +17,11 @@ function getCheckoutDestination( dependencies ) {
 		return '/checkout/' + dependencies.siteSlug;
 	}
 
+	/* NUX Trampoline A/B */
+	if ( 'trampoline' === abtest( 'nuxTrampoline' ) ) {
```

Comment:
This should be `getABTestVariation`, so that any users not assigned already in the `user` step don't get added at this point.